rachellegelden/Home-Together-Canada
# HTC-146: added new api key for .travis.yml

# [I146](https://github.com/rachellegelden/Home-Together-Canada/issues/146)

## Summary
Fixed api key (you cannot generate using cmd) so that's why the last build of develop failed to deploy

## Relevant Motivation & Context
Please provide relevant motivation and context for this change

## Testing Instructions
Merge, ensure this build passes, and then merge https://github.com/rachellegelden/Home-Together-Canada/pull/150 to test deployment

## Developer checklist prior to opening this pull request:


## Reviewer
